pkg/ignore/ignore_test.go

```diff
@@ -27,6 +28,33 @@ func TestIgnore_Match(t *testing.T) {
 	}
 }
 
+// Test all default ignore files, except for .git/info/exclude, since
+// that uses a .git directory that we cannot check in.
+func TestIgnoreDefaultIgoreFiles_Match(t *testing.T) {
+	// Change into testdata directory for this test
+	// since paths are relative
+	err := os.Chdir("testdata")
+	if err != nil {
+		panic(err)
+	}
+
+	i := NewIgnore([]string{"*.FROMARGUMENT"})
+	assert.NotNil(t, i)
+
+	assert.False(t, i.Match("notfoo"))
+	assert.True(t, i.Match("test.FROMARGUMENT")) // From .gitignore
+	assert.True(t, i.Match("test.DS_Store"))     // From .gitignore
+	assert.True(t, i.Match("test.IGNORE"))       // From .ignore
+	assert.True(t, i.Match("test.WOKEIGNORE"))   // From .wokeignore
+	assert.False(t, i.Match("test.NOTIGNORED"))  // From .notincluded - making sure only default are included
+
+	// Change directory back to original directory
+	err = os.Chdir("..")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Probably best to put this in the testing `Cleanup` method https://pkg.go.dev/testing#T.Cleanup and I'd recommend moving it up before the tests

Reply:
Also, please use the assert package

Reply:
Same as above, I apologize I didn't know about Cleanup, thanks for the pointer. Updated the code, please re-check.